AutomatedTesting/Gem/PythonTests/Performance/CMakeLists.txt

```diff
@@ -10,30 +10,37 @@ if(NOT PAL_TRAIT_BUILD_TESTS_SUPPORTED OR NOT PAL_TRAIT_BUILD_HOST_TOOLS)
     return()
 endif()
 
-ly_add_pytest(
-    NAME AutomatedTesting::EditorLevelLoadingPerfTests_DX12
-    TEST_SUITE periodic
-    TEST_REQUIRES gpu
-    TEST_SERIAL
-    PATH ${CMAKE_CURRENT_LIST_DIR}/TestSuite_Periodic_DX12.py
-    RUNTIME_DEPENDENCIES
-        Legacy::Editor
-        AZ::AssetProcessor
-        AutomatedTesting.Assets
-    COMPONENT
-        Performance
-)
+o3de_pal_dir(pal_dir ${CMAKE_CURRENT_LIST_DIR}/Platform/${PAL_PLATFORM_NAME} ${gem_restricted_path} ${gem_path} ${gem_parent_relative_path})
```

Comment:
To be safe we should surround each of the `${gem_...}` variables in double quotes to avoid empty variables from causing an incorrect number of parameters  to be supplied